Cannot find name 'STPPaymentMethod'.

After updating to latest version (5.3.0) I get an error:

```
ERROR in node_modules/nativescript-stripe/index.d.ts(100,29): error TS2304: Cannot find name 'STPPaymentMethod'.
node_modules/nativescript-stripe/index.d.ts(131,29): error TS2304: Cannot find name 'STPPaymentIntent'.
```

--- a/demo-angular/app/demo/creditcardview.component.ts
+++ b/demo-angular/app/demo/creditcardview.component.ts
@@ -1,5 +1,6 @@
 import { ChangeDetectorRef, Component } from "@angular/core";
 import { CreditCardView, PaymentMethod, Stripe, Token } from "nativescript-stripe";
+import { isAndroid } from "tns-core-modules/platform";
 import { publishableKey } from "./stripe.service";
 
 @Component({
@@ -28,6 +29,10 @@ export class CreditCardViewComponent {
   }
 
   createPaymentMethod(cardView: CreditCardView): void {
+    if (isAndroid) {
+      this.payment = "On Android this call cannot yet be called from UI thread";
+      return;
+    }
     this.payment = "Fetching payment method...";
     this.stripe.createPaymentMethod(cardView.card, (error, pm) => {
       this.payment = error ? error.message : this.formatPaymentMethod(pm);

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -97,7 +97,7 @@ export declare class PaymentMethod implements PaymentMethodCommon {
   customerId: string;
   metadata: object;
 
-  static fromNative(native: STPPaymentMethod): PaymentMethod;
+  static fromNative(native: any): PaymentMethod;
 }
 
 export declare interface StripePaymentIntentCommon {
@@ -128,7 +128,7 @@ export declare class StripePaymentIntent implements StripePaymentIntentCommon {
   requiresAction: boolean; // true if status == RequiresAction
   status: StripePaymentIntentStatus;
 
-  static fromNative(native: STPPaymentIntent): StripePaymentIntent;
+  static fromNative(native: any): StripePaymentIntent;
   static fromApi(json: any): StripePaymentIntent;
 }
 

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -158,7 +158,7 @@ export declare class StripeRedirectSession {
   native: any;
   readonly state: StripeRedirectState;
 
-  constructor(paymentIntent: StripePaymentIntent, completion: (clientSecret: string, error: Error) => void);
+  constructor(paymentIntent: StripePaymentIntent, cb: (error: Error, clientSecret: string) => void);
 
   startRedirectFlow(view: View): void;
   cancel(): void;

--- a/src/package.json
+++ b/src/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-stripe",
-  "version": "5.3.0",
+  "version": "5.3.1",
   "description": "NativeScript Stripe sdk",
   "main": "stripe",
   "typings": "index.d.ts",

--- a/src/stripe.ios.ts
+++ b/src/stripe.ios.ts
@@ -17,26 +17,15 @@ export class Stripe {
     const apiClient = ios.getter(STPAPIClient, STPAPIClient.sharedClient);
     apiClient.createTokenWithCardCompletion(
       card.native,
-      (token: STPToken, error: NSError) => {
-        if (!error) {
-          if (typeof cb === 'function') {
-            const newToken: Token = {
-              id: token.tokenId,
-              bankAccount: token.bankAccount,
-              card: card, // token.card is incomplete
-              created: new Date(token.created),
-              livemode: token.livemode,
-              android: null,
-              ios: token
-            };
-            cb(null, newToken);
-          }
-        } else {
-          if (typeof cb === 'function') {
-            cb(new Error(error.localizedDescription), null);
-          }
-        }
-      }
+      callback(cb, (token) => <Token>{
+        id: token.tokenId,
+        bankAccount: token.bankAccount,
+        card: card, // token.card is incomplete
+        created: new Date(token.created),
+        livemode: token.livemode,
+        android: null,
+        ios: token
+      })
     );
   }
 
@@ -63,17 +52,7 @@ export class Stripe {
     const params = STPPaymentMethodParams.paramsWithCardBillingDetailsMetadata(cardParams, billing, null);
     apiClient.createPaymentMethodWithParamsCompletion(
       params,
-      (pm: STPPaymentMethod, error: NSError) => {
-        if (!error) {
-          if (typeof cb === 'function') {
-            cb(null, PaymentMethod.fromNative(pm));
-          }
-        } else {
-          if (typeof cb === 'function') {
-            cb(new Error(error.localizedDescription), null);
-          }
-        }
-      }
+      callback(cb, (pm) => PaymentMethod.fromNative(pm))
     );
   }
 
@@ -81,17 +60,7 @@ export class Stripe {
     const apiClient = ios.getter(STPAPIClient, STPAPIClient.sharedClient);
     apiClient.retrievePaymentIntentWithClientSecretCompletion(
       clientSecret,
-      (pi: STPPaymentIntent, error: NSError) => {
-        if (!error) {
-          if (typeof cb === 'function') {
-            cb(null, StripePaymentIntent.fromNative(pi));
-          }
-        } else {
-          if (typeof cb === 'function') {
-            cb(new Error(error.localizedDescription), null);
-          }
-        }
-      }
+      callback(cb, (pi) => StripePaymentIntent.fromNative(pi))
     );
   }
 
@@ -101,19 +70,26 @@ export class Stripe {
     params.returnURL = returnUrl;
     apiClient.confirmPaymentIntentWithParamsCompletion(
       params,
-      (pi: STPPaymentIntent, error: NSError) => {
-        if (!error) {
-          if (typeof cb === 'function') {
-            cb(null, StripePaymentIntent.fromNative(pi));
-          }
-        } else {
-          if (typeof cb === 'function') {
-            cb(new Error(error.localizedDescription), null);
-          }
-        }
-      }
+      callback(cb, (pi) => StripePaymentIntent.fromNative(pi))
     );
   }
+}
+
+function callback(
+  cb: (error: Error, value: any) => void,
+  cvt: (value: any) => any):
+  (value: any, err: NSError) => void {
+  return (value: any, error: NSError) => {
+    if (!error) {
+      if (typeof cb === 'function') {
+        cb(null, cvt(value));
+      }
+    } else {
+      if (typeof cb === 'function') {
+        cb(new Error(error.localizedDescription), null);
+      }
+    }
+  };
 }
 
 export class Card implements CardCommon {
@@ -506,8 +482,11 @@ export class StripeRedirectSession {
   native: STPRedirectContext;
   readonly state: StripeRedirectState;
 
-  constructor(paymentIntent: StripePaymentIntent, completion: (clientSecret: string, error: NSError) => void) {
-    this.native = STPRedirectContext.alloc().initWithPaymentIntentCompletion(paymentIntent.native, completion);
+  constructor(paymentIntent: StripePaymentIntent, cb: (error: Error, clientSecret: string) => void) {
+    this.native = STPRedirectContext.alloc().initWithPaymentIntentCompletion(
+      paymentIntent.native,
+      callback(cb, (clientSecret) => clientSecret)
+    );
   }
 
   startRedirectFlow(view: View): void {

